src/transformers/trainer.py

```diff
@@ -441,9 +444,13 @@ def __init__(
                 self.backward_prefetch = BackwardPrefetch.BACKWARD_POST
 
             self.forword_prefetch = False
-            if "forword_prefetch" in self.args.fsdp_config and self.backward_prefetch:
+            if "forword_prefetch" in self.args.fsdp_config and self.forword_prefetch:
                 self.forword_prefetch = True
```

Comment:
This test can never be satisfied. Maybe you meant `self.args.fsdp_config.get("forword_prefect", False)`? (would also be nice to fix the typo)